parm/params_grib2_tbl_new

```diff
@@ -812,6 +812,16 @@
    3 192    36   0 SBTA177
    3 192    37   0 SBTA178
    3 192    38   0 SBTA179
+   3 192    79   0 SBTA1810
```

Comment:
@EricJames-NOAA Can you add these new grib2 names in parm/params_grib2_tbl_new.text which is the un-sorted grib2 table. You may refer to the table at https://github.com/NOAA-EMC/NCEPLIBS-g2tmpl/blob/e1397436a1584180fca755103e65ebce98d14619/src/params_grib2_tbl_new.text#L1318
We would like to keep these sorted and un-sorted tables consistent.